api/src/pcapi/sandboxes/scripts/save_sandbox.py

```diff
@@ -28,12 +28,17 @@ def save_sandbox(name: str, with_clean: bool = True) -> None:
 # The reason it is acceptable is that we only use this on the sandbox, which contains a small amount of data
 def _index_all_offers() -> None:
     logger.info("Reindexing offers")
+
     query = (
-        offer_models.Offer.query.with_entities(offer_models.Offer.id)
-        .filter(offer_models.Offer.isActive.is_(True))
-        .order_by(offer_models.Offer.id)
+        offers_models.Offer.query.outerjoin(offers_models.Stock)
```

Comment:
```suggestion
        db.session.query(offers_models.Offer).outerjoin(offers_models.Stock)
```
(low) On préfère `db.session.query` maintenant

Reply:
ah yes je change